core/build/build.odin

```diff
@@ -0,0 +1,233 @@
+package build
+
+import "core:strings"
+import "core:fmt"
+import "core:os"
+import "core:c/libc"
+import "core:path/filepath"
+import "core:sync"
+import "core:encoding/json"
+
+syscall :: proc(cmd: string, echo: bool) -> int {
```

Comment:
`syscall` associates with x86 instruction to call into kernel functions... maybe call this `shell_exec`?

Reply:
Will change the name, thanks. I was planning to document the package of course, but I wanted to get it going first and get some feedback on the general API